### DISCUSSION TITLE
personal-development-azar-1402/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#28] [Announcements]
# personal-development-azar-1402/

خب. آخر ماه شد و طبق روال ماه‌های قبل، گزارش این ماه رو منتشر می‌کنم.
متمم تو متمم اکثراً درس‌های تصمیم‌گیری و مهارت حل مسئله رو خوندم.
می‌خوام زودتر پروژه‌هاشون رو انجام بدم که بتونم تفکر استراتژیک رو ببینم چیه. به نظر خیلی چیز جالبی میاد.
فریب‌خورده تصادف نسیم طالب آدم جالبی هست. کتابش رو دوست دارم چون متفاوت فکر می‌کنه و دید جالبی نسبت به احتمالات و شانس داره.

https://aprd.ir/personal-development-azar-1402/

**Comment by alidalvand** [comment]
1.روز هم خواب نمی‌خوندم ؟؟؟
2.ببینم می‌تونم از ماه بعد یه برنامه‌ای برای بریزم یا نه؟؟؟

**Reply by pourmand1376** [reply]
ممنون از تذکرتون. تصحیح شد. 🙂

**Comment by mreleeam** [comment]
سلام مهندس جان.میدونم جاش اینجا نیست ولی اگر ممکنه منو راهنمایی کن❤️
من ۱۸ سالمه و ۳ سالی میشه شبکه خوندم و فیلد اصلیم SDWAN بوده.و قسمتی از سیلابس هاش توسعه پروگرم ها برای اتومیشن شبکه و سیستم تشخیص خطا روی استک AI بود "AI Driven SDWAN" بود و باید یک سری مباحث از ML رو مطالعه میکردم کم کم ازش خوشم اومد الان ۱ ماهیه که توی کلم زده کلا از شبکه بیام بیرون برم AI.
سر همین اصلا به درسام نرسیدم و اگر خیلی هنر کنم مهندسی کامپیوتر قزوین (شهر خودم) قبول بشم
حالا سوالم اینه که به نظر شما از ترم ۱ شروع کنم برای المپیاد دانشجویی بخونم یا برای کنکور ارشد؟

**Reply by pourmand1376** [reply]
سلام. وقتت بخیر. 

چند روزی هست سؤالت جلوی چشممه و خب نمی‌دونم چی جواب بدم. الان هم مطمئن نیستم چیزهایی که می‌نویسم جوابت باشه ولی چند کلمه‌ای می‌نویسم. فقط حتماً از بقیه هم مشورت بگیر؛ چون ممکنه با جواب من ذهنت بایاس بشه.

الان که می‌نویسم به نظرم میاد که المپیاد و کنکور و این بازی‌ها برای این بودند که آدم بتونه وارد یک دانشگاهی بشه که بعدش بتونه از قبل اون دانشگاه و ارتباطاتی که ایجاد کرده، کار بدست بیاره. وگرنه دانش خود سیستم‌عامل و شبکه و الگوریتم و این‌ها شاید مستقیماً تو کاری که می‌کنیم، تأثیر نداشته باشند و به درد نخورند. 

همون المپیاد و کنکور رو هم من از ترم یک به فکرش نبودم؛ یعنی اصلاً به ارشد خوندن فکر نمی‌کردم. یادمه ترم چهارم یا پنجم بود که یکی از دوستانم به نام جلال بهم گفت که:
- امیر بیا برو المپیاد شرکت کن. هیچی نیست المپیاد. 
- حاجی ولمون کن. این مسخره‌بازیا به من نیومده. حوصله ندارم. 
- تو اصلاً بیا شرکت کن. هیچی نمی‌خواد بخونی. فقط ثبت‌نام کن. بقیه‌اش با من. 

اینه که این دوستم هی رو مغز من کار کرد و با اصرار او المپیاد شرکت کردم و خب داستان‌ رفت جلو. 

الان هم فکر می‌کنم این که آدم از ترم یک بخواد راجع به المپیاد و کنکور فکر کنه، چندان مسیر بهینه‌ای نیست. اون‌ها رو باید در یکی دو سال آخر کارشناسی بهشون فکر کرد. 

حالا وارد دانشگاه میشی بیشتر می‌بینی. خوشبختانه جو رشته کامپیوتر خیلی کاریه. بچه‌ها دنبال نمره جمع‌کردن نیستند. دنبال این هستند که مهارت بدست بیارن و از مهارت‌شون پول بدست بیارند. این که از الان یه مهارتی داری خیلی خوبه. خیلی بهتره که بتونی تو این حوزه یا هر حوزه‌ دیگه‌ای که دوست داری متخصص بشی. این از هر المپیاد و کنکوری بهتره. 

می‌دونم تقریباً هیچی نگفتم و احتمالاً هم حرفام کمکی بهت نمی‌کنه. ولی اینهایی که گفتم تنها چیزهایی هستند که فعلاً بهشون فکر می‌کنم. 

این مطلب از محمدرضا هم شاید بهت کمک کنه تو تصمیمت:
- [بهره برداری از ظرفیت ها و گوسفندنگری به قابلیت‌ها - روزنوشته‌های محمدرضا شعبانعلی](https://mrshabanali.com/%D8%A8%D9%87%D8%B1%D9%87-%D8%A8%D8%B1%D8%AF%D8%A7%D8%B1%DB%8C-%D8%A7%D8%B2-%D8%B8%D8%B1%D9%81%DB%8C%D8%AA-%D9%87%D8%A7-%D9%88-%D9%86%DA%AF%D8%B1%D8%B4-%DA%AF%D9%88%D8%B3%D9%81%D9%86%D8%AF%D8%A7%D9%86/)
